src/js/stores/MesosFetchers.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
```

Comment:
Can we name the file so it somehow resembles the source of the data (summary store)?